lesson_06/homework_06.py

```diff
@@ -22,12 +33,28 @@
 + напишіть цикл for що перебере і обробить всі значення списку alien_color
 """
 
+alien_color = ["green", "yellow", "red"]
+for color in alien_color:
+    if color == "green":
+        print("Player gained 5 points")
+    elif color == "red":
+        print("Player gained 15 points")
+    else:
+        print("Player gained 10 points")
+
 # task 5
 """  Начинки для піци (pizza_topping): напишіть цикл, який пропонує користувачеві ввести ряд начинок
 для піци, доки він не введе значення 'quit'. Коли вони введуть кожну начинку,
 надрукуйте повідомлення про те, що ви додасте цю начинку до їхньої піци.
 """
 
+while True:
+    pizza_topping = input("Введіть начинку для піци: ")
+    if pizza_topping == "quit":
+        break
+    else:
+        print("Начинку до вашої піци додано!")
```

Comment:
```suggestion
        print(f"Начинку {pizza_topping} до вашої піци додано!")
```
Більш дружній вивід )